argon-doc-viewer.css

```diff
@@ -52,68 +45,69 @@ header {
 
 /* Common Formatting */
 
+#summary {
+  padding: 4px 0;
+}
+
 #summary marked-element::shadow #content pre {
   background-color: #fafafa;
+  border: solid #ebebeb;
+  border-width: 1px 0;
   overflow-x: auto;
-  padding: 12px 16px;
+  padding: 12px 18px;
 }
 
 #summary marked-element::shadow #content code {
   @apply(--paper-font-code1);
 }
 
-/* Property Sections */
-
-#properties,
-#methods {
-  border: 1px solid;
-  margin: 16px 0;
-}
-#properties > header,
-#methods > header {
-  color: #ffffff;
-  padding: 16px;
+#summary marked-element::shadow #content p {
+  padding: 0 18px;
 }
 
-#properties {
-  background-color: rgba(255, 86, 33, 0.2);
-}
-#properties > header {
-  background-color: rgb(255, 86, 33);
-}
-#properties,
-#properties iron-doc-property {
-  border-color: rgb(255, 86, 33);
+#summary marked-element::shadow #content a {
+  color: black;
 }
 
-#methods {
-  background-color: rgba(0, 190, 164, 0.2);
+#summary marked-element::shadow #content h1,
+#summary marked-element::shadow #content h2,
+#summary marked-element::shadow #content h3,
+#summary marked-element::shadow #content h4,
+#summary marked-element::shadow #content h5 {
+  padding: 0 18px;
 }
-#methods > header {
-  background-color: rgb(0, 190, 164);
+
+/* Property Sections */
+
+.card {
+  @apply(--shadow-elevation-2dp);
+  background: white;
+  margin-bottom: 20px;
 }
-#methods,
-#methods iron-doc-property {
-  border-color: rgb(0, 190, 164);
+
+.card > header {
+  @apply(--paper-font-title);
+  padding: 18px 16px;
+  border-bottom: 1px solid #ebebeb;
```

Comment:
Since we're already depending on `paper-styles` in this non-paper element, it would be preferable to try to make use of the custom property colors found there, where reasonable.